grim.go

```diff
@@ -218,3 +222,9 @@ func (i *Instance) checkGrimQueue() error {
 
 	return nil
 }
+
+func BuildTruncatedMessage(truncateID string) string {
```

Comment:
This function doesnt need to be exported.  If it is exported it needs to be commented.  golint would tell you this so I suspect you are not running golint?  (make check will do this for you)